ovs/extensions/db/arakoon/ArakoonInstaller.py

```diff
@@ -172,6 +172,29 @@ def __init__(self):
         raise RuntimeError('ArakoonInstaller is a complete static helper class')
 
     @staticmethod
+    def archive_existing_arakoon_data(ip, directory, top_dir, cluster_name):
+        new_client = SSHClient(ip, username='ovs')
+        logger.debug('archive - check if {0} exists'.format(directory))
+        if new_client.dir_exists(directory):
+            logger.debug('archive - from {0}'.format(directory))
+            archive_dir = top_dir + 'archive/' + cluster_name
+            if new_client.dir_exists(archive_dir + '/' + os.path.basename(directory)):
+                logger.debug('archive - from existing archive {0}'.format(archive_dir))
+                timestamp = time.strftime('%Y-%m-%dT%H:%M:%S', time.gmtime())
+                new_archive_dir = archive_dir + '-' + timestamp
+                new_client.dir_create(new_archive_dir)
+                new_client.run('/bin/mv {0} {1}'.format(archive_dir, new_archive_dir))
```

Comment:
It's tricky to use ```/bin/mv``` as paths tend to change between Ubuntu and CentOS. I think just ```mv``` is better here